app/views/sessions/new.html.erb

```diff
@@ -1,2 +1,37 @@
-<h1>Sessions#new</h1>
-<p>Find me in app/views/sessions/new.html.erb</p>
+<style>
```

Comment:
This is fine for now.

Css and javascript should be written in `app/assets`

CSS is applied based on an order of priority: 
- Inline stles
- In page styles
- external stylesheets

This is an example of an in page style. It will take precedence over any external styles making your page harder to maintain. All styles should be written as external stylesheets